packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsApprovalFlow/approvalStepUtils.ts

```
@@ -0,0 +1,92 @@
+
+export const mapApprovalStatusToPayableStatus = (
+  status: components['schemas']['ApprovalRequestStatus']
+): components['schemas']['PayableStateEnum'] => {
+  switch (status) {
+    case 'approved':
+      return 'waiting_to_be_paid';
+    case 'rejected':
+      return 'rejected';
+    case 'canceled':
+      return 'canceled';
+    case 'waiting':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)